src/InfluxDB.Collector/Configuration/CollectorBatchConfiguration.cs

```diff
@@ -4,6 +4,6 @@ namespace InfluxDB.Collector.Configuration
 {
     public abstract class CollectorBatchConfiguration
     {
-        public abstract CollectorConfiguration AtInterval(TimeSpan interval);
+        public abstract CollectorConfiguration AtInterval(TimeSpan interval, int? maxBatchSize = null);
```

Comment:
Should we default this to a value in line with what InfluxDB suggests? (1000 to 5000?)

Reply:
Can we add an overload of this method that accepts the `maxBatchSize` instead, so that we can avoid bumping the major version? (Binary-incompatible change)